.github/workflows/auto_tagger.yml

```diff
@@ -0,0 +1,68 @@
+name: Auto tag new actions
+
+on:
+  pull_request:
+    branches: [master]
+    types: [opened, synchronize, reopened, closed]
+
+jobs:
+  find_changes:
+    name: "Find changed directories that need tagging"
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.get-matrix.outputs.matrix }}
+      matrix_empty: ${{ steps.get-matrix.outputs.matrix_empty }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - id: get-matrix
+        uses: smartlyio/find-changes-action@v1
+        with:
+          directory_containing: action.yml
+
+  create_tag:
+    name: "Create tags"
+    runs-on: ubuntu-20.04
+    needs: find_changes
+    if: needs.find_changes.outputs.matrix_empty == 'false' && github.event_name == 'pull_request'
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
Is it safe to tag in a matrix build like this?

If we ever want to have typescript actions in this repository, it may be an issue. Those have a build step that needs to be committed before tagging.

Reply:
Result: we can process this later